healthtools/config.py

```diff
@@ -17,8 +17,13 @@
     # Clinical document endpoint
     "cloudsearch_cos_endpoint": "http://doc-cfa-healthtools-ke-cos-nhxtw3w5goufkzram4er7sciz4.eu-west-1.cloudsearch.amazonaws.com/",
     # Health facilities endpoint
-    "cloudsearch_health_faciities_endpoint":"https://doc-health-facilities-ke-65ftd7ksxazyatw5fiv5uyaiqi.eu-west-1.cloudsearch.amazonaws.com",
+    "cloudsearch_health_faciities_endpoint": "https://doc-health-facilities-ke-65ftd7ksxazyatw5fiv5uyaiqi.eu-west-1.cloudsearch.amazonaws.com",
```

Comment:
@DavidLemayian funny. I did this on the other branch..I merged to develop on my local repo and continued with the slack integration. This was handled